src/styles/index.css

```diff
@@ -132,6 +133,7 @@ body {
   left: -13px;
   -webkit-transform: rotate(90deg);
   transform: rotate(90deg);
+  cursor: pointer;
```

Comment:
It's a good practice to group related CSS properties together. Move the 'cursor' property up to be next to the 'transform' property for better organization. https://9elements.com/css-rule-order/
Fix it everywhere